org.OpenT2T.Onboarding.Nest/org.OpenT2T.Onboarding.Nest.xml

```diff
@@ -1,27 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
-<!--  This is the metadata for a category of things that are connected via Nest. -->
-<node>
-    <interface name="com.OpenT2T.Onboarding.Nest">
```

Comment:
Perhaps I'm missing some context but why is the nest interface prefixed with com. while the wink is prefixed with org.?